stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputViewStyle.kt

```diff
@@ -271,27 +271,27 @@ public data class MessageInputViewStyle(
 
                 val pictureAttachmentIcon = a.getDrawable(
                     R.styleable.MessageInputView_streamUiPictureAttachmentIcon
-                ) ?: context.getDrawableCompat(R.drawable.stream_ui_attachment_permission_media)!!
+                ) ?: context.getDrawableCompat(R.drawable.stream_ui_attachment_tab_media)!!
```

Comment:
Also, renaming this resource might be breaking, but IDK if we ever had a deprecation process for resources

Reply:
I renamed the icons back just in case somebody overrides them.